src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/DeprecationControl.xaml

```diff
@@ -0,0 +1,76 @@
+<UserControl x:Class="NuGet.PackageManagement.UI.DeprecationControl"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:nuget="clr-namespace:NuGet.PackageManagement.UI"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             mc:Ignorable="d"
+             d:DesignHeight="150" d:DesignWidth="350">
+  <UserControl.Resources>
+    <ResourceDictionary>
+      <ResourceDictionary.MergedDictionaries>
+        <nuget:SharedResources />
+      </ResourceDictionary.MergedDictionaries>
+      <BitmapImage
+        x:Key="BitmapImage_StatusWarning"
+        UriSource="/NuGet.PackageManagement.UI;component/Resources/statuswarning.16.16.png" />
+    </ResourceDictionary>
+  </UserControl.Resources>
+  <Grid>
+    <Expander
+      Margin="0,12,0,16"
+      IsExpanded="True"
+      Foreground="{DynamicResource {x:Static nuget:Brushes.UIText}}">
+      <Expander.Header>
+        <StackPanel
+          Orientation="Horizontal"
+          VerticalAlignment="Center">
+          <TextBlock
+            FontWeight="Bold"
+            Text="{x:Static nuget:Resources.Label_Deprecated}"/>
+          <Image
+            Margin="8,0,0,0"
+            Height="16"
+            Width="16"
+            Source="{StaticResource BitmapImage_StatusWarning}"/>
+        </StackPanel>
+      </Expander.Header>
+      <StackPanel
+        Orientation="Vertical">
+        <TextBlock
+          Margin="0,8,0,0"
+          TextWrapping="WrapWithOverflow"
+          TextTrimming="WordEllipsis"
+          Text="{Binding Path=PackageDeprecationReasons}" />
```

Comment:
Have you checked how narrator handles this? 
You mind need to add automation properties if it doesn't. 

There's ample examples throughout the code.

Reply:
Applied some improvements in https://github.com/NuGet/NuGet.Client/pull/2899/commits/92ae10d9471130c8bc5a635695c80546d6acbbb6.